python/requirements.txt

```diff
@@ -26,6 +26,7 @@ numpy>=1.20
 pyarrow >= 9.0.0
 
 # ray[all]
+boto3==1.38.27
```

Comment:
Add `boto3` to install packages before building ray. So that, it could support s3-compatible storage.